### PR TITLE
Oculus Mobile fixes

### PR DESCRIPTION
**Pull request by mcclure**
* Corrections to the Android getting started page, re: name scheme of the org.lovr.test app
* Too many monkeys. Reduce the number of monkeys

Should be merged into 0.13.0 and master

### CODE REVIEW

File: examples/Instancing/main.lua [hunk 1]
```diff
@@ -3,6 +3,10 @@
 function lovr.load()
   MONKEYS = 1000
 
+  if lovr.headset.getDriver() ~= "oculusmobile" then -- That's too many monkeys!!!
```
Comment (bjornbytes) [inline examples/Instancing/main.lua]: Wait is this supposed to be `==`?

Comment (bjornbytes) [reply examples/Instancing/main.lua]: Feel free to change to 700 for everyone too

Comment (bjornbytes) [reply examples/Instancing/main.lua]: Should be enough monkeys for anyone

Comment (mcclure) [reply examples/Instancing/main.lua]: It is supposed to be == :( Very confused how I made this mistake

Comment (mcclure) [reply examples/Instancing/main.lua]: one second will fix

**Comment by bjornbytes** [comment]
Thanks, will cherrypick elsewhere

**Pull request merged**